.mailmap

```diff
@@ -0,0 +1 @@
+Reuben Nixon-Hill <reuben.nixon-hill10@imperial.ac.uk> <ReubenHill@users.noreply.github.com>
```

Comment:
Is this because you did some "bad" commits in the past?

Reply:
Name and email change after I got married - forgot to add it when I last committed to FInAT master.